chained_filter_controller/test/test_chained_filter.cpp

```diff
@@ -0,0 +1,93 @@
+#include <gtest/gtest.h>
```

Comment:
```suggestion
#include <gmock/gmock.h>
```
We are using gmock consistently, see https://github.com/ros-controls/ros2_controllers/pull/1625